EnglishFilter/sections/antiadblock.txt

```diff
@@ -544,6 +544,8 @@ kzclip.com,ru-clip.com,ruclip.com###morev
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/73781
+escapegames24.com#%#//scriptlet("prevent-addEventListener", "load")
```

Comment:
It's too wide. Please exchange to the `escapegames24.com#%#//scriptlet("prevent-setTimeout", "ads", "2000")`.
Also add the rule below to the `content_blocker.txt` (below common rules like `blockadblock` and `g207` sections).
```
! https://github.com/AdguardTeam/AdguardFilters/issues/73781
@@||escapegames24.com^$generichide
```

Reply:
Should I add at that two places? Or just one?

Reply:
What do you mean by wide rule even?

Reply:
It will prevent all `eventListener` with `load`. Without third parameter it's wide.